### ISSUE TITLE
Bitwarden massively degrading Chrome's performance

### ISSUE THREAD

**Issue by quthla** [bitwarden/mobile#628]
Ever since I switched from iOS to Android, I'm having huge issues when browsing big websites. It will load to like 75% and then massively stutter until it freezes completely, depending on how big the website is.

I've suspected Bitwarden a long time already, in particular the accessibility service, but turning it off didn't improve the situation. Only after I turned off both services, the autofill and accessibility service, the issue was gone.

OnePlus 6T, Android Pie

[This](https://github.com/docker/cli/blob/master/contrib/completion/zsh/_docker) is an example of a website which makes Chrome freeze for me.

**Comment by kspearrin** [comment]
I was able to reproduce the problem, but it occurs for me even if I have both autofill services disabled.

**Comment by quthla** [comment]
You must force close chrome after disabling both services.

**Comment by kspearrin** [comment]
Thanks, that worked.

This might be an issue with Chrome itself. I tried LastPass's autofill service and can reproduce the same issue.

I tried the same website with Firefox using Bitwarden Autofill Service and the problem does not occur.

**Comment by quthla** [comment]
Quite many issues with Chrome apparently. Have you reported any of the issues to the Chrome bugtracker maybe so they can fix or look into them?

**Comment by kspearrin** [comment]
I opened the issue on Chrome's bug tracker here: https://bugs.chromium.org/p/chromium/issues/detail?id=1014945

**Comment by quthla** [comment]
That's good news. Let's hope they'll look into it.

And the other issues like autofill stops working until browser restarted etc? I have a 100% safe way to reproduce that within seconds

**Comment by kspearrin** [comment]
Feel free to open another issue for that.

**Comment by quthla** [comment]
Here you mean? The issue has already been discussed thoroughly

**Comment by quthla** [comment]
https://github.com/bitwarden/mobile/issues/489 is what I'm talking about

**Comment by quthla** [comment]
Are you sure you don't mean the issue with accessiblity service that you only read Chrome's app id instead ot the site url?

What I mean is that the autofill service randomly stops working until Chrome is restarted.
1. Browse any login page with Chrome
2. Check if the autofill dropdown is showing
3. Open a link in an app which embeds Chrome like Slack for example
4. Go back to Chrome - autofill will not work anymore until restarted

**Comment by kspearrin** [comment]
> 
> 
> Are you sure you don't mean the issue with accessiblity service that only read Chrome's app id instead of the site url?
> 
> What I mean is that the autofill service randomly stops working until Chrome is restarted.
> 
>     1. Browse any login page with Chrome
> 
>     2. Check if the autofill dropdown is showing
> 
>     3. Open a link in an app which embeds Chrome like Slack for example
> 
>     4. Go back to Chrome - autofill will not work anymore until restarted


Could you download LastPass and see if you are able to reproduce the same issue with LastPass's autofill service? I am having trouble getting LastPass to work on my emulator.

**Comment by djsmith85** [comment]
Hi @quthla, it appears we've not heard from you about this issue in some time. While we are now closing this issue, if it continues to be a problem please feel free to re-open this issue and provide any further information that may be helpful in reproducing the problem or helping you further.
